### PR TITLE
Add symfony 7 support

### LINKED ISSUE #49: Incompatible with Symfony 7.1
Hi,

When trying to update to Sulu 2.6.4 and Symfony 7.1 I'm running into a rather awkward situation. When just updating `sulu/sulu symfony/*` composer prevents the update. This is due to "sulu/web-twig v2.5.6 conflicts with symfony/property-access v7.1.1.". Additional "fun fact" is that this conflict has only be declared in version 2.5.5+ of this package (by #45), meaning that a plain `composer update` (or `composer update sulu/* symfony/*`) will happily _downgrade_ this package to 2.5.4 which doesn't have this conflict defined yet. But assuming that the conflict was declared due to some incompatibility it is my believe that with a downgraded version of this package that incompatibility will still be present (+ downgrading obviously means that the bugfixes of later versions of this package are not applied anymore).

### PR DESCRIPTION
**Pull request by alexander-schranz**
fixes #49

### CODE REVIEW

File: docs/image.md [hunk 1]
```diff
@@ -130,6 +130,34 @@ This could be:
...
+         title="Description"
+         src="/uploads/media/sulu-400x400/01/image.jpg?v=1-0"
+         class="image-class">
+</picture>
+```
+
+This could be:
+
+```twig
+{{ get_image(image,
```
Comment (Prokyonn) [inline docs/image.md:153]: The example does not match with the above one, I think it makes more sense if we show the exact output of this call

File: src/ImageExtension.php [hunk 2]
```diff
@@ -100,7 +100,7 @@ public function __construct(
         array $defaultAttributes = [],
         array $defaultAdditionalTypes = [],
         bool $aspectRatio = false,
-        array $imageFormatConfiguration = null
+        ?array $imageFormatConfiguration = null,
```
Comment (Prokyonn) [inline src/ImageExtension.php:103]: This will not work with php < 8.0 (trailing comma)

**Review by Prokyonn — CHANGES_REQUESTED** [review]

File: docs/image.md (hunk 1 above)
Comment (alexander-schranz) [reply docs/image.md:145]: good catch. updated the example output

File: phpstan-baseline.neon [hunk 3]
```diff
@@ -0,0 +1,151 @@
+parameters:
```
Comment (alexander-schranz) [inline phpstan-baseline.neon:1]: update phpstan to `^1`

**Review by Prokyonn — APPROVED** [review]

**Pull request merged**